pom.xml

```diff
@@ -16,11 +16,11 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.11</maven.compiler.source>
```

Comment:
Java 11 brings new language features, but it has a disadvantage. Classes compiled for jdk 11 cannot be used in java 8 runtime, while java 8 is target. If you want to have `var` feature, please add `lombok` dependency (you can use `val` that's even better), but only for test classes (the intention was to have the exact source and decompiled lines for library code)